Gems/ROS2/Code/Platform/Common/Clang/ros2_editor_static_clang.cmake

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+set(LY_COMPILE_OPTIONS
+        PRIVATE
+            -fexceptions
+)
```

Comment:
You will need to include this compiler specific file in the CMakeLists.txt PLATFOMR_INCLUDE section for the `ROS2.Editor` target in order to get it to be used.

```
PLATFORM_INCLUDE_FILES
        Platform/Common/${PAL_TRAIT_COMPILER_ID}/ros2_editor_static_${PAL_TRAIT_COMPILER_ID_LOWERCASE}.cmake
```